__std__/plutus-apps/packages/check-the-flake.nix

```diff
@@ -14,8 +14,9 @@ cell.library.pkgs.writeShellApplication {
     root="$(repo-root)"
 
     shell_fragments=$(
-      find \
-        "$root/__std__/cells" \
+      find . \
+        # TODO(std) rename to $root/nix when std'ization finished
```

Comment:
Like I did for plutus-core, I'm keeping both `nix-shell` and `nix develop` for a couple weeks so that if `nix develop` breaks we can use `nix-shell` while I fix it. 

The very final PR will address this comment and will delete all old nix files